server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/qualitymodel/RatingSettings.java

```diff
@@ -93,10 +91,10 @@ public long getDevCost(@Nullable String languageKey) {
 
   private long getDefaultDevelopmentCost() {
     try {
-      return Long.parseLong(settings.getString(DEVELOPMENT_COST));
+      return Long.parseLong(config.get(DEVELOPMENT_COST).get());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3655)